src/App.js

```diff
@@ -1,23 +1,23 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
+//import logo from './logo.svg';
 import './App.css';
 
 class App extends Component {
   render() {
     return (
       <div className="App">
         <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
+          <h1>Who Twote That </h1>
           <p>
-            Edit <code>src/App.js</code> and save to reload.
+            A Twitter game to see how well you know your friends, or really any two Twitter users for that matter. I'm using this to learn <a href="https://reactjs.org/tutorial/tutorial.html">React.js</a>.
           </p>
```

Comment:
Any idea why git hates these tabs? Would recommend using `prettier` to get auto-formatting down so you don't have to worry about Windows doing weird things to your files

Reply:
I already have it installed actually, so I will run it and hopefully it fixes things.